R-package/R/lgb.Booster.R

```diff
@@ -790,12 +788,23 @@ Booster <- R6::R6Class(
 #'               the values in \code{params} take precedence.
 #' @param ... ignored
 #' @return For prediction types that are meant to always return one output per observation (e.g. when predicting
-#'         \code{type="response"} on a binary classification or regression objective), will return a vector with one
-#'         element per row in \code{newdata}.
+#'         \code{type="response"} or \code{type="raw"} on a binary classification or regression objective), will
+#'         return a vector with one element per row in \code{newdata}.
 #'
 #'         For prediction types that are meant to return more than one output per observation (e.g. when predicting
-#'         \code{type="response"} on a multi-class objective, or when predicting \code{type="leaf"}, regardless of
-#'         objective), will return a matrix with one row per observation in \code{newdata} and one column per output.
+#'         \code{type="response"} or \code{type="raw"} on a multi-class objective, or when predicting \code{type="leaf"},
+#'         regardless of objective), will return a matrix with one row per observation in \code{newdata} and one column per output.
+#'
+#'         For \code{type="leaf"} predictions, will return a matrix with one row per observation in \code{newdata}
+#'         and one column per tree. Note that for multiclass objectives, LightGBM trains one tree per class. That means
```

Comment:
```suggestion
#'         and one column per tree. Note that for multiclass objectives, LightGBM trains one tree per class at each boosting iteration. That means
```

Reply:
good point! I agree we should add this.

Reply:
pushed these changes in https://github.com/microsoft/LightGBM/pull/5384/commits/b3eb3205086b790dedcba9d4a89057fdc5032220

(did that instead of accepting them in the browser since the roxygen docs needed to be regenerated)